### PR TITLE
Updates to ROSA ingress-testing

### PR DESCRIPTION
**Pull request by krishvoor**
This PR, updates the config used.

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml [hunk 1]
```diff
@@ -81,7 +81,12 @@ tests:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: nightly
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      CONFIG: config/classic-rosa.yml
```
Comment (svetsa-rh) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml:85]: @krishvoor I think you'll need to add support to override this env variable in https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-commands.sh

**Review by svetsa-rh — COMMENTED** [review]
CONFIG env should be added for 4.13 nightly test as well.

**Comment by svetsa-rh** [comment]
@krishvoor It looks like the pod on prow where ingress test is running, doesn't have access to config/classic-rosa.yml.

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml:1]: @svetsa-rh is going to remove this config file.

**Comment by qiliRedHat** [comment]
The following 'Result parsing failed' logs looks not correct in job
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41697/rehearse-41697-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/1684435559663013888/artifacts/perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/openshift-qe-ingress-perf/build-log.txt
```
time="2023-07-27 06:36:37" level=info msg="Tool:wrk termination:http servers:45 concurrency:18 procs:1 connections:200 duration:5m0s" file="runner.go:89"
time="2023-07-27 06:37:07" level=info msg="Running sample 1/5: 5m0s" file="exec.go:82"
time="2023-07-27 06:38:33" level=error msg="Result parsing failed, skipping: unexpected end of JSON input" file="exec.go:132"
```

**Comment by qiliRedHat** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-stable-prod-perfscale-rosa-multiaz-ingress-perf-defaultnodes

**Comment by krishvoor** [comment]
> The following 'Result parsing failed' logs looks not correct in job https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41697/rehearse-41697-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/1684435559663013888/artifacts/perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/openshift-qe-ingress-perf/build-log.txt But the error was just for 1/5 and 2/5, 3/5-5/5 were fine. Let me trigger a new test to check if this is a flack
> 
> ```
> time="2023-07-27 06:36:37" level=info msg="Tool:wrk termination:http servers:45 concurrency:18 procs:1 connections:200 duration:5m0s" file="runner.go:89"
> time="2023-07-27 06:37:07" level=info msg="Running sample 1/5: 5m0s" file="exec.go:82"
> time="2023-07-27 06:38:33" level=error msg="Result parsing failed, skipping: unexpected end of JSON input" file="exec.go:132"
> ```

I see, we hit the same [issues](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41697/rehearse-41697-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/1684490124827561984/artifacts/perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/openshift-qe-ingress-perf/build-log.txt) , will push a change to run in `LOG_LEVEL:DEBUG`
as suggested here: https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/ingress-perf/run.sh#L7C1-L7C10

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic

**Comment by krishvoor** [comment]
The [ingress-job](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41697/rehearse-41697-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/1684607823897432064/artifacts/perfscale-rosa-multiaz-ingress-perf-3nodes-periodic/openshift-qe-ingress-perf/build-log.txt) successfully completed

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-defaultnodes-periodic

**Comment by krishvoor** [comment]
The [recent failure](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/41697/rehearse-41697-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-defaultnodes-periodic/1685916348439334912) is not related to the PR, will retrigger this.

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-perfscale-rosa-multiaz-ingress-perf-defaultnodes-periodic

**Pull request merged**